src/backend/web/routes/feeds.js

```diff
@@ -52,6 +52,27 @@ feeds.get('/:id', async (req, res) => {
   }
 });
 
+feeds.put('/:id/flag', protectAdmin(true), async (req, res) => {
```

Comment:
Remove the `true` in `protectAdmin()`, that's only if we want to redirect a user vs. JSON route.